### ISSUE TITLE
How to blacklist mob of mods?

### ISSUE THREAD

**Issue by Wilco90** [Globox1997/RpgDifficulty#4]
Hello, I was wondering if you could explain to me how I can fix my crashes.
I have mods active like outvoted, and zombified and some other ones with more mob varients, but they crash my game and I have 0 coding skills. 
	I changed this in the config but it does not work Example: I tried this: ,outvoted.wildfire
	Then I changed it to ,outvoted:wildfire, but it changed the text in the file to this.. I tried some other things but I can not figure out how to exclude mobs.
// Exluded Entity List Bsp: minecraft.villager
	"excluded_entity": [
		"outvoted",
		":wildfire",
		"outvoted",
		":glutton",
		"outvoted",
		":barnacle",
		"outvoted",
		":meerkat",
		"outvoted",
		":ostrich",
		"zombified",
		":arctic_zombie",
		"zombified",
		":beserker_zombie",
		"zombified",
		":crystal_zombie",
		"zombified",
		":infernal_zombie",
		"zombified",
		":miner_zombie",
		"zombified",
		":sludge_zombie"
	],

**Comment by Globox1997** [comment]
yes it is only one dot between the modid and mobid so like you first tried "outvoted.wildfire"
what is important here, the modid and mobid has to be correct
For example, in the fabric.mod.json you can find the correct modid like here: [https://github.com/How-Bout-No/Outvoted/blob/arch-1.17/fabric/src/main/resources/fabric.mod.json#L3](https://github.com/How-Bout-No/Outvoted/blob/arch-1.17/fabric/src/main/resources/fabric.mod.json#L3)
and here the mobid: [https://github.com/How-Bout-No/Outvoted/blob/06c257f2cfef8355a3166ac6164a753b760173ca/common/src/main/java/io/github/how_bout_no/outvoted/init/ModEntityTypes.java#L16](https://github.com/How-Bout-No/Outvoted/blob/06c257f2cfef8355a3166ac6164a753b760173ca/common/src/main/java/io/github/how_bout_no/outvoted/init/ModEntityTypes.java#L16)
So basically your first intention was correct
